spec.html

```diff
@@ -6418,7 +6417,7 @@
           1. Return *true*, if every field in _Desc_ is absent.
           1. Return *true*, if every field in _Desc_ also occurs in _current_ and the value of every field in _Desc_ is the same value as the corresponding field in _current_ when compared using the SameValue algorithm.
           1. If the [[Configurable]] field of _current_ is *false*, then
-            1. Return *false*, if the [[Configurable]] field of _Desc_ is *true*.
+            1. Return *false*, if the [[Configurable]] field of _Desc_ is present and its value is *true*.
```

Comment:
I've often wondered why it wasn't done like this. I guess there is no reason, so thanks for fixing :)